src/components/FeaturedContent/FeaturedContent.stories.tsx

```diff
@@ -0,0 +1,351 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { withDesign } from "storybook-addon-designs";
+import Button from "../Button/Button";
+import Heading from "../Heading/Heading";
+import SimpleGrid from "../Grid/SimpleGrid";
+
+import FeaturedContent, {
+  featuredContentWidthArray,
+  featuredContentPositionArray,
+} from "./FeaturedContent";
+
+const meta: Meta<typeof FeaturedContent> = {
+  title: "Components/Basic Elements/FeaturedContent",
+  component: FeaturedContent,
+  decorators: [withDesign],
+  argTypes: {
+    className: { control: false },
+    id: { control: false },
+    isFullWidth: {
+      table: { defaultValue: { summary: false } },
+    },
+    imageProps: { disable: false },
+    "imageProps.alt": {
+      control: { type: "text" },
+      table: { defaultValue: { summary: "" } },
+    },
+    "imageProps.width": {
+      control: { type: "select" },
+      options: featuredContentWidthArray,
+      table: { defaultValue: { summary: "oneHalf" } },
+    },
+    "imageProps.position": {
+      control: { type: "select" },
+      options: featuredContentPositionArray,
+      table: { defaultValue: { summary: "end" } },
+    },
+    "imageProps.src": {
+      table: { defaultValue: { summary: "" } },
+    },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof FeaturedContent>;
+
+/**
+ * Main Story for the FeaturedContent component. This must contain the `args`
+ * and `parameters` properties in this object.
+ */
+
+export const WithControls: Story = {
+  args: {
+    className: undefined,
+    id: "FeaturedContent-id",
+    isFullWidth: false,
+    textContent: undefined,
+    "imageProps.alt": "Alt text",
+    "imageProps.position": "end",
+    "imageProps.width": "default",
+    "imageProps.src": "//placekitten.com/600/600",
+  },
+  render: (args) => (
+    <FeaturedContent
+      {...args}
+      textContent={
+        <div>
+          <Heading level="h1" overline="Featured">
```

Comment:
Sorry, I missed this before. Update the `"h1"` to `"h2"` for all the examples. In an app, the "h1" is for the page title rather than the heading here, and it should only be used once on a page. Using "h2" is safer but it also depends on the app and it's heading hierarchy.